CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ Changelog is rather internal in nature. See release notes for the public overvie
 
 ## Version 2.0.0
 
+- [#465]
+  - **Description:** KDropdownMenu improvements & bind $attrs in KIconButton
+  - **Products impact:** -
```

Comment:
I think this change would fall into "updated API" products impact category since there are some updates to public facing API for components like a new slot and attributes binding